app.js

```diff
@@ -282,7 +282,9 @@
       },
 
       getStringQuery: function(type) {
-        return new RegExp(this.$('.query.' + type).val(), 'i');
+        var query = this.$('.query.' + type).val();
+        var escapedQuery = query.replace(/[-[\]{}()*+?.,\\^$|#\s]/g, "\\$&");
```

Comment:
Although I didn't plan it this way, I think so; I think it won't match if it doesn't escape the space too. 
https://gyazo.com/c0683cb6a92f4d253640f1434ceefeb7